tests/scripts/all.sh

```diff
@@ -2572,14 +2572,56 @@ psa_crypto_config_accel_all_curves_except_one () {
     make test
 }
 
+# The 2 following components can be seen as a single one with the goal to
+# accelerate all curves while disabling RSA and run tests.
+# Keep in sync with test_psa_crypto_config_reference_curves() since these
+# are used for driver's coverage analysis in "analyze_outcomes.py".
 component_test_psa_crypto_config_accel_all_curves_except_p192 () {
     psa_crypto_config_accel_all_curves_except_one MBEDTLS_ECP_DP_SECP192R1_ENABLED
 }
-
 component_test_psa_crypto_config_accel_all_curves_except_x25519 () {
     psa_crypto_config_accel_all_curves_except_one MBEDTLS_ECP_DP_CURVE25519_ENABLED
 }
 
+# Reference component to be used in conjunction with:
+# - component_test_psa_crypto_config_accel_all_curves_except_p192
+# - component_test_psa_crypto_config_accel_all_curves_except_x25519
+component_test_psa_crypto_config_reference_curves () {
+    msg "build: PSA_CRYPTO_CONFIG + USE_PSA_CRYPTO"
+
+    # Disable ALG_STREAM_CIPHER and ALG_ECB_NO_PADDING to avoid having
+    # partial support for cipher operations in the driver test library.
+    scripts/config.py -f include/psa/crypto_config.h unset PSA_WANT_ALG_STREAM_CIPHER
+    scripts/config.py -f include/psa/crypto_config.h unset PSA_WANT_ALG_ECB_NO_PADDING
+
+    # full config (includes USE_PSA, TLS 1.3 and driver support)
```

Comment:
I think these lines up to "Run the tests" are common to the reference and driver components. I suggest we factor them into a config functions like we did for the other driver/ref pairs, unless you see a reason not to.